bindings/python/multibody/fcl/expose-fcl.cpp

```diff
@@ -18,89 +13,8 @@ namespace pinocchio
-        .value("BV_UNKNOWN",::hpp::fcl::BV_UNKNOWN)
-        .value("BV_AABB",::hpp::fcl::BV_AABB)
-        .value("BV_OBB",::hpp::fcl::BV_OBB)
-        .value("BV_RSS",::hpp::fcl::BV_RSS)
-        .value("BV_kIOS",::hpp::fcl::BV_kIOS)
-        .value("BV_OBBRSS",::hpp::fcl::BV_OBBRSS)
-        .value("BV_KDOP16",::hpp::fcl::BV_KDOP16)
-        .value("BV_KDOP18",::hpp::fcl::BV_KDOP18)
-        .value("BV_KDOP24",::hpp::fcl::BV_KDOP24)
-        .value("GEOM_BOX",::hpp::fcl::GEOM_BOX)
-        .value("GEOM_SPHERE",::hpp::fcl::GEOM_SPHERE)
-        .value("GEOM_CAPSULE",::hpp::fcl::GEOM_CAPSULE)
-        .value("GEOM_CONE",::hpp::fcl::GEOM_CONE)
-        .value("GEOM_CYLINDER",::hpp::fcl::GEOM_CYLINDER)
-        .value("GEOM_CONVEX",::hpp::fcl::GEOM_CONVEX)
-        .value("GEOM_PLANE",::hpp::fcl::GEOM_PLANE)
-        .value("GEOM_HALFSPACE",::hpp::fcl::GEOM_HALFSPACE)
-        .value("GEOM_TRIANGLE",::hpp::fcl::GEOM_TRIANGLE)
-        .value("GEOM_OCTREE",::hpp::fcl::GEOM_OCTREE)
-        .value("NODE_COUNT",::hpp::fcl::NODE_COUNT)
-        ;
-      }
-      
-      if(!eigenpy::register_symbolic_link_to_registered_type< ::hpp::fcl::Contact>())
-      {
-        ContactPythonVisitor::expose();
-      }
-      if(!eigenpy::register_symbolic_link_to_registered_type< std::vector< ::hpp::fcl::Contact> >())
-      {
-        StdVectorPythonVisitor<ContactPythonVisitor::Contact>::expose("StdVec_Contact");
-      }
-      
-      if(!eigenpy::register_symbolic_link_to_registered_type< ::hpp::fcl::CollisionResult>())
-      {
-        CollisionResultPythonVisitor::expose();
-      }
-      if(!eigenpy::register_symbolic_link_to_registered_type< std::vector< ::hpp::fcl::CollisionResult> >())
-      {
-        StdVectorPythonVisitor<CollisionResultPythonVisitor::CollisionResult>::expose("StdVec_CollisionResult");
-      }
-      
-      if(!eigenpy::register_symbolic_link_to_registered_type< ::hpp::fcl::DistanceResult>())
-      {
-        DistanceResultPythonVisitor::expose();
-      }
-      if(!eigenpy::register_symbolic_link_to_registered_type< std::vector< ::hpp::fcl::DistanceResult> >())
-      {
-        StdVectorPythonVisitor<DistanceResultPythonVisitor::DistanceResult>::expose("StdVec_DistanceResult");
-      }
-      
-      if(!eigenpy::register_symbolic_link_to_registered_type< ::hpp::fcl::CollisionGeometry>())
-      {
-        CollisionGeometryPythonVisitor::expose();
-      }
-      
-      typedef ::hpp::fcl::MeshLoader MeshLoader;
-      typedef ::hpp::fcl::CachedMeshLoader CachedMeshLoader;
-
-      if(!eigenpy::register_symbolic_link_to_registered_type<MeshLoader>())
-        MeshLoaderPythonVisitor<MeshLoader>::
-        expose("Class to create CollisionGeometry from mesh files.");
-
-      if(!eigenpy::register_symbolic_link_to_registered_type<CachedMeshLoader>())
-        MeshLoaderPythonVisitor<CachedMeshLoader>::
-        expose("Class to create CollisionGeometry from mesh files with cache mechanism.");
-      
```

Comment:
The implicit conversion, a few lines below, never gets called. This causes a unit test failure. @jcarpent, any idea why ?

      bp::implicitly_convertible< Transform3f,SE3 >();

Reply:
@jmirabel You should keep the symbolink link to MeshLoader and so on.

Reply:
I don't understand your answer. If I have to keep all the `if(!eigenpy::register_symbolic_link_to_registered_type) { ... }`, then this PR is useless...

Reply:
This is why the tests are not passing because pinocchio.MeshLoader() does not exist. Because it is inside the scope hppfcl. I'm forcing that here for convenience.

Reply:
I realized that and updated the examples and the unit test. The only thing which I do not know how to fix now is the conversion from SE3 to Transform3f.

Reply:
I'm not able to see it in the test.
But again, I think important import hppfcl Meshloader, geometric entities inside Pinocchio scope. It will make life easy and will not breack the Python current API.

Reply:
> I'm not able to see it in the test.

Ah, the failure on my computer and on CI is not the same... I just pushed a fix of the CI failure. On my computer, I still have a failure on bindings_fcl_transform.py

Reply:
@jmirabel Please keep shortcut to not break the current Python API.